plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
```

Comment:
I've merged all of the sub modules into check_mongodb.py.  We can break it out again if we end up writing more checks that could benefit from them.  I do think the nagios module could benefit from being re-written as an enum like I did for the mongo replset statuses, but that can be something else we improve later.